src/transformers/models/gpt2/modeling_gpt2.py

```diff
@@ -198,7 +198,7 @@ def _attn(self, query, key, value, attention_mask=None, head_mask=None):
         if not self.is_cross_attention:
             # if only "normal" attention layer implements causal mask
             query_length, key_length = query.size(-2), key.size(-2)
-            causal_mask = self.bias[:, :, key_length - query_length : key_length, :key_length].bool()
+            causal_mask = self.bias[:, :, key_length - query_length : key_length, :key_length].to(torch.bool)
```

Comment:
Needed to be able to TorchScript the traced model, this should not break things because it is equivalent, from the [docs](https://pytorch.org/docs/stable/generated/torch.Tensor.bool.html):

> self.bool() is equivalent to self.to(torch.bool). See [to()](https://pytorch.org/docs/stable/generated/torch.Tensor.to.html#torch.Tensor.to).